src/applications/static-pages/authenticated-homepage/components/App/App.js

```diff
@@ -0,0 +1,41 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { selectShowAuthenticatedHomepage } from 'applications/static-pages/authenticated-homepage/selectors';
+import environment from 'platform/utilities/environment';
+
+export const App = ({ showAuthHomePage }) => {
+  if (environment.isProduction) {
+    return null;
+  }
+
+  if (!showAuthHomePage) {
+    return null;
+  }
+
+  document.getElementById('homepage').style.display = 'none';
```

Comment:
Might want to do something like `?.style?.display` in case it's not able to find that node 🙂

Reply:
Good point, but that throws an error: 
![image](https://user-images.githubusercontent.com/14869324/91510644-ef8fae80-e89a-11ea-805f-054bed20b336.png)

Reply:
TIL “ Optional chaining not valid on the left-hand side of an assignment” https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
This will likely create a flicker showing the unauth content when the user is authenticated, because the JS bundle has to be downloaded first before the `document.getElementById('homepage').style.display = 'none';`

Reply:
Ah got it, never knew that! Thanks Erik! Perhaps you could do something like this then? Just trying to make sure there isn't a runtime error when we can't find the homepage in case someone were to ever accidentally change the ID without knowing this is dependent on it 🙂 

```js
const homepage = document.getElementById('homepage');

if (homepage) {
  homepage.style.display = 'none';
}
```